src/EnergyPlus/HVACInterfaceManager.cc

```diff
@@ -888,29 +901,25 @@ void ManageSingleCommonPipe(EnergyPlusData &state,
     Real64 CommonPipeTemp;
 
     auto &PlantCommonPipe(state.dataHVACInterfaceMgr->PlantCommonPipe);
-    auto &MyEnvrnFlag(state.dataHVACInterfaceMgr->MyEnvrnFlag_SingleCommonPipe);
 
     // One time call to set up report variables and set common pipe 'type' flag
-    if (state.dataHVACInterfaceMgr->OneTimeData_SingleCommonPipe) {
```

Comment:
These had extra flags and arrays just to make sure two separate Array1Ds got allocated to the right length one time, rather than just simply adding a single boolean to the struct.  This is much better now.